apps/packages/ui/src/components/Option/ResearchWorkspace/WorkspaceHeader.tsx

```
@@ -548,6 +591,187 @@ export const WorkspaceHeader: React.FC<WorkspaceHeaderProps> = ({
+      setPersonaProfiles(personas)
+    } catch {
+      if (requestId !== defaultAssistantRequestIdRef.current) return
+      resetDefaultAssistantModalState()
+      setDefaultAssistantError(
+        t(
+          "playground:workspace.defaultAssistantLoadError",
+          "Could not load default assistant settings."
+        )
+      )
+    } finally {
+      if (requestId !== defaultAssistantRequestIdRef.current) return
```

Biome (2.5.0)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)